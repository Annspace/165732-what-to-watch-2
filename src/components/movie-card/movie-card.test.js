import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';
import mockData from '../../mocks/films';

it(`renders correctly`, () => {
  const hoverCardHandler = jest.fn();
  const MovieCardComponent = renderer
    .create(<MovieCard
      id={mockData.movies[0].id}
      posterImage={mockData.movies[0].posterImage}
      title={mockData.movies[0].title}
      onHoverCard={hoverCardHandler}/>);
  expect(MovieCardComponent).toMatchSnapshot();
});
