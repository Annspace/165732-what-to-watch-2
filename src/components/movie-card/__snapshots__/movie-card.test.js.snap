// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseLeave={[MockFunction]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <video
      height="175"
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films-1"
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
